src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -61,10 +71,16 @@ const RegisterForm = () => {
       userName: '',
       userEmail: '',
       password: '',
+      repeatPassword: '',
     },
     validationSchema: schema,
     validateOnChange: false,
     onSubmit: (values, { resetForm }) => {
+      if (values.password !== values.repeatPassword) {
+        // Set error for repeatPassword field
+        formik.setFieldError('repeatPassword', 'Passwords do not match');
+        return; // Stop submission if passwords don't match
+      }
       dispatch(register(values));
 
       localStorage.setItem(
```

Comment:
I don't think it is a good idea to save user data to localstorage in terms of security issues.